code/src/components/Movies.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable linebreak-style */
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Movies = ({ movieList }) => {
+  console.log(movieList)
+  return (
+    <section>
+      <h2 className="header-text"> UPCOMING MOVIES </h2>
+      <div className="movies-wrapper">
+        {movieList.map((movie) => {
+          return (
+            <Link
+              key={movie.id}
+              to={`/moviedetails/${movie.id}`}>
+              <div className="card-info">
+                <h1 className="movietitle"> {movie.title} </h1>
```

Comment:
Using both className="class-name" and className="classname" might cause some confusion